meta/main.yml

```diff
@@ -0,0 +1,28 @@
+galaxy_info:
+  author: Nic Henke <nic.henke@versity.com>
+  description: A really decent guy.
```

Comment:
Pretty much the first thing I say when I mention you to friends.

Reply:
I stole it from you.